Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -445,7 +445,7 @@ def _parse_hit(self, root_hit_elem, query_id):
             yield hit
 
     def _parse_hsp(self, root_hsp_frag_elem, query_id, hit_id):
-        """Iterator that transforms Hit_hsps XML elements into HSP objects.
+        """Transform Hit_hsps XML elements into HSP objects (PRIVATE).
```

Comment:
I'd like the docstring to indicate this is a generator (iterator), not sure how best to rephrase this though.

Reply:
What about `Yield a generator object that transforms Hit_hsps XML elements into HSP objects.`